cmd/codegen/main.go

```diff
@@ -66,10 +68,17 @@ func ClientGen(cmd *cobra.Command, args []string) error {
 	if moduleName != "" {
 		cfg.ModuleName = moduleName
 
-		if moduleContextPath == "" {
-			return fmt.Errorf("--module-name requires --module-context")
+		if modulePath == "" {
+			return fmt.Errorf("--module-name requires --module-path")
 		}
-		cfg.ModuleContextPath = moduleContextPath
+		modulePath, err = relativeTo(outputDir, modulePath)
+		if err != nil {
+			return err
+		}
+		if strings.HasPrefix(modulePath, "..") || strings.HasPrefix(modulePath, "../") {
```

Comment:
nit, just informational: `filepath.IsLocal` is convenient in these situations

Reply:
Ah, we can't use that here since we only want to check if it's a parent - however, I did simplify this to using `strings.Cut` to avoid the repetition here.